sci-physics/lhapdf/lhapdf-6.3.0-r1.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Maybe we want to bump it to `EAPI=8` since we might have to hang on to this version for a while until python is fixed in the 6.5.X series.

Reply:
Yes, I initial tried sth. similar to the 6.5.X EAPI=8 ebuilds for 6.3.0, but since the python interface/wrapper changed it did not work out of the box.